packages/contracts-bedrock/src/L1/ETHLockbox.sol

```diff
@@ -1,9 +1,10 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.25;
+pragma solidity 0.8.15;
 
 // Contracts
-import { ProxyAdminOwnedBase } from "src/L1/ProxyAdminOwnedBase.sol";
 import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+import { ProxyAdminOwnedBase } from "src/L1/ProxyAdminOwnedBase.sol";
+import { ReinitializableBase } from "src/universal/ReinitializableBase.sol";
```

Comment:
At some point we should just enable import sorting, will be a large diff though https://book.getfoundry.sh/reference/config/formatter#sort_imports

Reply:
will make an issue to trakc this